src/linter.rs

```diff
@@ -398,10 +400,10 @@ impl Linter {
   }
 }
 
-pub trait Plugins {
+pub trait Plugin {
   fn run(
     &mut self,
     context: &mut Context,
     program: swc_ecmascript::ast::Program,
-  );
+  ) -> Result<(), AnyError>;
```

Comment:
Use [`anyhow::Error`](https://docs.rs/anyhow/1.0.32/anyhow/struct.Error.html) instead

Reply:
`deno_core` uses anyhow@v1.0.32, while `deno_lint` uses v1.0.34. Is it better to use v1.0.32 in `deno_lint` too?

Reply:
@magurotuna sure, sounds good

Reply:
ah I was mistaken, both `deno_core` and `deno_lint` specify anyhow's version in caret style (by default), and `1.0.32` and `1.0.34` are compatible in semver - so there's no problem with current Cargo.toml